src/volatile_set.h

```diff
@@ -0,0 +1,40 @@
+#ifndef VOLATILESET_H
+#define VOLATILESET_H
+
+#include <stddef.h>
+#include "rax.h"
+#include "sds.h"
+
+typedef struct {
+    sds (*entryGetKey)(const void *entry);
+
+    long long (*getExpiry)(const void *entry);
+
+    int (*expire)(void *entry);
+
+} volatileEntryType;
+
+
+typedef struct {
+    volatileEntryType *etypr;
+    rax *expiry_buckets;
```

Comment:
nit - I always associate `buckets` with hashtables but I assume this is not the same `buckets`?
```suggestion
    rax *entries_by_expiry;
```

Reply:
@PingXie I will comment here (as this repeated in some earlier comments as well). this is just a stub implementation and I would suggest avoid even reviewing that part. it's just not worth the time as most of the work was done in https://github.com/ranshid/valkey/pull/4 and https://github.com/ranshid/valkey/pull/5

Reply:
sg